tools/lang_batch_translate.py

```
@@ -0,0 +1,192 @@
+                print(f"    → {locale}: 已完整")
+
+    # Step 4: 整合所有缺失 key 进行批量翻译
+    print("\n=== Step 4: 整合缺失 key 并翻译 ===")
+    trans_maps = {}
+
+    with ThreadPoolExecutor(max_workers=3) as executor:
+        futures = {}
+        for locale in TARGET_LOCALES:
+            all_missing_texts = []
+            for module_info in missing_by_locale[locale].values():
+                # 这里简化收集，需要翻译的文本（你可以根据需要扩展）
```

Ruff (0.15.14)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)